test/wrappers/vroom_test.rb

```diff
@@ -791,4 +791,19 @@ def test_quantity_precision
       assert_equal 1.001, activity[:detail][:quantities].first[:value]
     }
   end
+
+  def test_partially_nil_capacities
+    problem = VRP.basic
+    problem[:services].each{ |service|
+      service[:quantities] = [{ unit_id: 'kg', value: 1 }]
+    }
+    problem[:vehicles].first[:capacities] = [{ unit_id: 'kg', limit: 3 }]
+
+    result = OptimizerWrapper.wrapper_vrp('demo', { services: { vrp: [:vroom] }}, TestHelper.create(problem), nil)
+
+    assert_operator result[:routes].first[:activities].count{ |activity| activity[:service_id] }, :<=, 3,
+                    'The vehicle cannot load more than 3 services and 3 kg'
+
+    assert_empty result[:unassigned], 'The second vehicle has no capacity limit and should the remaining services'
```

Comment:
```suggestion
    assert_empty result[:unassigned], 'The second vehicle has no capacity limit and should be assigned to the remaining services'
```

Reply:
There is only 1 vehicle in this test, and 3 services. There is no second vehicle. Shouldn't we have more than 3 services and one additional vehicle ?

Reply:
Oops, I thought we were in the previous configuration of the above test (11 services and 2 vehicles)